app/packages/utilities/src/size-bytes-estimate.ts

```diff
@@ -0,0 +1,68 @@
+const OBJECT_CLASS_CACHE = {};
+
+const objectClass = (object: object) => {
+  const prototype = Object.prototype.toString.call(object);
+  if (!OBJECT_CLASS_CACHE[prototype]) {
+    OBJECT_CLASS_CACHE[prototype] = prototype.slice(8, -1);
+  }
+  return OBJECT_CLASS_CACHE[prototype];
+};
+
+const isObject = (object: object) => {
+  const oClass = objectClass(object);
+  return oClass === "Object" || oClass === "Array";
+};
+
+type SizeTypes = null | object | string | undefined;
+
+type SizerTypes = SizeTypes | Array<SizeTypes>;
+
+const sizer = (object: SizerTypes) => {
```

Comment:
Cleaner implementation of what currently exists
* https://github.com/voxel51/fiftyone/blob/7f73a783426922f986579ab8217dbda308a9de00/app/packages/looker/src/overlays/util.ts#L24